src/test/java/emissary/util/ShortNameComparatorTest.java

```diff
@@ -37,6 +37,15 @@ public void testNonsenseNames() {
         assertEquals("Bad components sort in alpha order", "foo-att-abc", l.get(0).shortName());
     }
 
+    @Test
+    public void testNonsenseNames2() {
```

Comment:
"Nonsense" is subjective; this test might be better named "testComparisonOfShortNamesThatHaveNoChildren" or a shorter variant